### ISSUE TITLE
[PLUS] - Not getting value for `isMyContact` field in JSON data

### ISSUE THREAD

**Issue by mehmetcoskun** [allburov/whatsapp-http-api#125]
### JSON Example
```json
{
    "id": "1234567890",
    "isBusiness": false,
    "isEnterprise": false,
    "name": "John Doe",
    "shortName": "John",
    "type": "in",
    "isBlocked": false
}
```

The JSON data above does not include the value for the `isMyContact` field. 

### Expected Behavior
I expect to receive the `isMyContact` value in the JSON data.

**Comment by allburov** [comment]
Hi! 
1. What is the payload for? Is it a "message" webhook data?
2. Does the field disappear or there always was no such field?
3. Do you use default engine?

**Comment by mehmetcoskun** [comment]
Hi!

1. It is a /api/contacts/all response
2. The JSON data does not include the `isMyContact` field at all.
3. Current Engine: WEBJS

**Comment by allburov** [comment]
Hm, I'm not sure but I believe the endpoint show only your contacts, doesn't it?
I'll look at this anyway a bit later!

**Comment by mehmetcoskun** [comment]
No, it also shows people in the group who are not in my contacts, until the previous update everything was working, there was a value like isMyContact but now there is no value and it would be nice if there was a parsing like isGroup

**Comment by allburov** [comment]
Got it, thank you for the details!
